apps/s_time.c

```diff
@@ -240,16 +224,10 @@ int s_time_main(int argc, char **argv)
                                    www_path);
             if (buf_len <= 0 || SSL_write(scon, buf, buf_len) <= 0)
                 goto end;
-            while ((i = SSL_read(scon, buf, sizeof(buf))) > 0 ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_READ ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_WRITE)
```

Comment:
Well, I re-based and squashed the s_time cleanup after your patch.
I expected at least the -www option to do real session renegotiation.
but that does probably not happen because the number of connects
per second seems to be no different with -reuse:

./openssl s_server -www

./openssl s_time -www /test -reuse
vs.
./openssl s_time -www /test -new